### PR TITLE
[SPARK-29504][WEBUI] Toggle full job description on click

### PR DESCRIPTION
**Pull request by PavithraRamachandran**
On clicking job description in jobs page, the description was not shown fully.
Add the function for the click event on description.

when there is a long description of a job, it cannot be seen fully in the UI.
The feature was added in https://github.com/apache/spark/pull/24145
But it is missed after https://github.com/apache/spark/pull/25374

Before change:
Screenshot from 2019-10-23 11-23-00
After change: on Double click over decription
Screenshot from 2019-10-23 11-20-02

No

Manually test

### CODE REVIEW

**Comment by PavithraRamachandran** [comment]
@gengliangwang i have raised for 2.4 branch. Could you verify?

File: core/src/main/resources/org/apache/spark/ui/static/webui.js [hunk 1]
```diff
@@ -83,4 +83,10 @@ $(function() {
   collapseTablePageLoad('collapse-aggregated-rdds','aggregated-rdds');
   collapseTablePageLoad('collapse-aggregated-activeBatches','aggregated-activeBatches');
   collapseTablePageLoad('collapse-aggregated-completedBatches','aggregated-completedBatches');
+});
```
Comment (gengliangwang) [inline core/src/main/resources/org/apache/spark/ui/static/webui.js]: Nit: add a blank line here.

Comment (PavithraRamachandran) [reply core/src/main/resources/org/apache/spark/ui/static/webui.js]: added the blank space as per the review comment. Could you verify

**Comment by gengliangwang** [comment]
Hi @PavithraRamachandran ,

After building and running test manually with the changes, I found it doesn't work well: on double click the class `description-input` is removed.
In the original PR https://github.com/apache/spark/pull/25484, the file `additional-metrics.js` is not removed. 
I am closing this PR now.
Sorry I didn't look close into https://github.com/apache/spark/pull/25484 .

**Pull request closed without merging**